contrib/bootstrap/README.md

```diff
@@ -0,0 +1,25 @@
+Generating Bootstrap
+====================
+
+We use bootstrap's customizer to ensure our bootstrap is the small and also
+stays with the aesthetics of debian.
+
+The file `config.json` contains related configuration.
+
+How To Generate
+-----------------
+
+- Go to https://getbootstrap.com/customize/
+
+- Upload the new `config.json` file, making sure you change the one in
+  `/contrib/bootstrap` to match.
+
+- Download the `bootstrap.zip` file
+  ```sh
+  BASE=/path/to/debsouces/repo # make sure this points to top level
+  mkdir tmp
+  unzip ~/Downloads/bootstrap.zip -d ./tmp # Download location may vary
```

Comment:
why not in /tmp ? this would simplify things (avoid creating a dir and then needing to remove it ...)